pkg/shell.go

```diff
@@ -14,6 +14,7 @@ import (
 const StdinBufferSize = 4096
 
 type ShellRequest struct {
+	ServiceId      utils.Id
```

Comment:
I would have named it `ServiceID` to respect current format (or rename others with `Id`)

Reply:
done